internal/presenters/presenter_local_finding.go

```diff
@@ -151,24 +151,37 @@ func (p *LocalFindingPresenter) Render() (string, error) {
 	mainTmpl := localFindingsTemplate.Lookup("main")
 
 	err = mainTmpl.Execute(buf, struct {
-		Summary        SummaryData               `json:"summary"`
-		Results        local_models.LocalFinding `json:"results"`
-		Order          []string
-		ShowIgnored    bool
-		SeverityFilter string
+		Summary         SummaryData               `json:"summary"`
+		Results         local_models.LocalFinding `json:"results"`
+		Order           []string
+		OpenFindings    []local_models.FindingResource
+		IgnoredFindings []local_models.FindingResource
+		ShowIgnored     bool
+		SeverityFilter  string
 	}{
-		Summary:        sum,
-		Results:        p.Input,
-		Order:          []string{"low", "medium", "high"},
-		ShowIgnored:    p.ShowIgnored,
-		SeverityFilter: p.SeverityMinLevel,
+		Summary:         sum,
+		Results:         p.Input,
+		OpenFindings:    filterOutIgnoredFindings(p.Input.Findings),
+		IgnoredFindings: p.Input.Findings,
```

Comment:
Issue: This is currently not used. We want to extend the test cases to cover when ignored issues should be included in the output.